dts/arc/quark_se_c1000_ss.dtsi

```diff
@@ -0,0 +1,53 @@
+#include "skeleton.dtsi"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "snps,arcem4";
+			reg = <1>;
+		};
+	};
+
+	flash0: flash@40010000 {
+		reg = <0x40000000 DT_FLASH_SIZE>;
+	};
+
+	sram0: memory@a8000400 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0xa8000400 DT_SRAM_SIZE>;
+	};
+
+	dccm0: dccm@80000000 {
+		device_type = "memory";
+		compatible = "intel,qmsi_ss-dccm";
```

Comment:
are these really properties specific to intel,qmsi? seem like the are arc related?

Reply:
i will correct it